src/2025h2/relink-dont-rebuild.md

```diff
@@ -1,10 +1,11 @@
 # Relink don't Rebuild
-| Metadata         |                        |
-| --------         | ---                    |
-| Point of contact | @yaahc                 |
-| Status           | Proposed               |
-| Tracking issue   | |
-| Zulip channel    | |
+| Metadata         |                                    |
+| :--              | :--                                |
+| Point of contact | @yaahc                             |
+| Status           | Proposed                           |
+| Tracking issue   | [rust-lang/rust-project-goals#400] |
+| Zulip channel    |                                    |
+
 
```

Comment:
The code isn't picking up champions when the sections are separated by empty lines like here. And that in turn makes the champion reports miss these goals of course.

(Also, shouldn't this goal be in the "faster builds" flagship themes?)